pkgs/games/nanosaur2/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, fetchFromGitHub, SDL2, cmake, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "nanosaur2";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jorio";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-UY+fyn8BA/HfCd2LCj5cfGmQACKUICH6CDCW4q6YDkg=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [
+    SDL2
+    makeWrapper
+  ];
+
+  configurePhase = ''
+    cmake -S . -B build -DCMAKE_BUILD_TYPE=Release
```

Comment:
Can this be converted to cmakeFlags? https://nixos.org/manual/nixpkgs/stable/#cmake
Something like:
```
  cmakeFlags = [ "-B build" "-DCMAKE_BUILD_TYPE=Release" ];
```

Reply:
No, while I can't tell you why, I can tell you this does *not* work. Also whatever we change here, we should also change at nanosaur.

Generally this is getting *frustrating*! All the pangea games are basically the same, and with each PR another person reviews, and has *opinions*, and want different changes, while IMHO it really should be *always the same*. Nanosaur, Bugdom, Otto Matrix, Cro Mag Rally, and now Nanosaur2, its really all the same, but it seems we need a different `.nix` file for each.

Reply:
Keep in mind that each reviewer is a completely independent person. There's enough of us that we can't really talk to each other, but each one of us wants to make things as standardised and up to date to the current best practices as possible. The previous nix files may be left without updates if they're not changed for other reasons, but any changes will get questions... and finally we may just miss something that's picked up by another person. Sorry you're frustrated about the process - we're all trying to do our best here.

To avoid common issues, you can for example run the nixpkgs-hammering and resolve them before submission. Which actually I should've done as well - there's some other fixes I'll add comments for as well.
(`nix run github:jtojnar/nixpkgs-hammering nanosaur2`)

Reply:
Yes, I'm very much frustrated with the process, not with you, and I apologize if I made you feel this way. I now updated both Nanosaur & Nanosaur2, so both should be better. Aligning all the Pangea packages, so they are all identical for maintainabillity should *probably* be done eventually, but not here I assume.